python/kserve/kserve/model_server.py

```diff
@@ -117,9 +117,10 @@ def start(self, models: Union[List[Model], Dict[str, Deployment]], nest_asyncio:
             # formula as suggest in https://bugs.python.org/issue35279
             self.max_asyncio_workers = min(32, utils.cpu_count()+4)
 
-        logging.info(f"Setting asyncio max_workers as {self.max_asyncio_workers}")
-        asyncio.get_event_loop().set_default_executor(
-            concurrent.futures.ThreadPoolExecutor(max_workers=self.max_asyncio_workers))
+        if self.workers == 1:
```

Comment:
@Suresh-Nakkeran just saw this [comment](https://github.com/tornadoweb/tornado/issues/2426#issuecomment-400895086), can you try removing worker check and moving the asyncio initialization code after the "fork"(line131) see if multi-processing works .

Reply:
you're right @yuzisun . multi-processing is working by moving initialization code after starting http server.
![image](https://user-images.githubusercontent.com/30799624/148900426-05766e2e-fad0-4506-9025-6f82da978ac4.png)